website/docs/docs/cloud/secure/vcs-privatelink.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Configuring PrivateLink for self-hosted cloud version control systems (VCS)"
+id: vcs-privatelink
+description: "Setting up a PrivateLink connection between dbt Cloud and an organization’s cloud hosted git server"
+sidebar_label: "PrivateLink for VCS"
+---
+
+import SetUpPages from '/snippets/_available-tiers-privatelink.md';
+
+<SetUpPages features={'/snippets/_available-tiers-privatelink.md'}/>
+
+AWS PrivateLink provides private connectivity from dbt Cloud to your self-hosted cloud version control system (VCS) service by routing requests through your virtual private cloud (VPC). This type of connection does not require you to publicly expose an endpoint to your VCS repositories or for requests to the service to traverse the public internet, ensuring the most secure connection possible. AWS recommends PrivateLink connectivity as part of it's [Well-Architected Framework](https://docs.aws.amazon.com/wellarchitected/latest/framework/welcome.html) and details this particular pattern in the **Shared Services** section of the [AWS PrivateLink whitepaper](https://docs.aws.amazon.com/pdfs/whitepapers/latest/aws-privatelink/aws-privatelink.pdf).
+
+This document will cover at a high level the resources necessary to implement this solution. Cloud environments and provisioning processes vary greatly, so information from this guide may need to be adapted to fit your requirements.
+
+## PrivateLink connection
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/privatelink-vcs-architecture.png" width="80%" title="High level overview of the dbt Cloud and AWS PrivateLink for VCS architecture" />
+
+### Solution resources
```

Comment:
It might be helpful to make this title more descriptive. Maybe something like:
```suggestion
### Creating the Interface VPC PrivateLink connection
```

Reply:
I hesitated to use this language originally because this isn't creating the connection. We do that, not them, they gather the resources and hand them to our team. How about
```suggestion
### Required resources for creating a connection
```

Reply:
I like that!